regulations/generator/api_reader.py

```diff
@@ -85,9 +85,16 @@ def layer(self, layer_name, doc_type, label_id, version=None):
             doc_id = root
         else:
             doc_id = '{}/{}'.format(version, root)
-        return self._get(
+        result = self._get(
             ('layer', layer_name, doc_type, root, str(version)),
             'layer/{}/{}/{}'.format(layer_name, doc_type, doc_id))
+        # To remove - also try the old format for CFR layers; the API may not
+        # have been updated
+        if result is None and doc_type == 'cfr':
+            result = self._get(
+                ('layer', layer_name, 'old', layer_name, root, str(version)),
```

Comment:
It was just a placeholder, but thinking about it again, there's no reason to not use the same key. Will update.